src/main/java/com/the/hugging/team/repositories/TransactionRepository.java

```diff
@@ -90,16 +90,16 @@ public void delete(Transaction obj) {
     }
 
     public List<Transaction> getByCashRegister(CashRegister cr) {
-        List<Transaction> cashRegisterSpecific = new LinkedList<>();
```

Comment:
Why are we using LinkedList instead of ArrayList?

Reply:
Because all other repos use LinkedList. We could change all of them later.